test/ecto/changeset_test.exs

```diff
@@ -614,6 +614,29 @@ defmodule Ecto.ChangesetTest do
 
   ## Validations
 
+  describe "validation_specs/1" do
+
+    test "returns an empty list if no validations are specified" do
+      result = 
+        changeset(%{})
+        |> validation_specs()
+      
+      assert result == []
+    end
+
+    test "returns the appropriate validations" do
+      result =
+        changeset(%{})
```

Comment:
i think that, according to the styleguidelines, the first param of the first function of a pipe should be on it's own line, so

```elixir
result =
  %{}
  |> changeset()
  |> ...
```

Reply:
I've changed my code, and the other 20 or so occurrences of this already there. 

However, in future I'd suggest the style guidelines be amended so that this only applies if the parameter is the actual starting value of the pipeline. In this case, the parameter was a default passed to the function.

Reply:
Yeah, I agree with you @pragdave here. This is one of the checks that credo does, but we have it disabled for ecto, because in many cases the starting value isn't really part of the pipeline (e.g. the data type of the first element is different from what you pipe through the other calls).